src/containers/ProfileDetails/components/SSID/index.js

```diff
@@ -332,17 +332,17 @@ const SSIDForm = ({
           mode === 'wpa3MixedEAP') && (
           <Item
             name="radiusServiceName"
```

Comment:
this needs to be changed to `radiusServiceId` also this id needs to be added/removed from `childProfileIds` for the profile